nft/nfts.yaml

```diff
@@ -155,8 +155,11 @@ paths:
                         $ref: '#/components/schemas/contractAddress'
                       tokenId:
                         $ref: '#/components/schemas/tokenId'
+                      tokenType:
+                        $ref: '#/components/schemas/tokenId'
```

Comment:
this should be `#/components/schemas/tokenId` right?

Reply:
whoop, ty